examples/recording_service/service_admin/cpp/CMakeLists.txt

```diff
@@ -16,13 +16,26 @@ project (RecorderRemoteAdmin)
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CONNEXTDDS_DIR}/resource/cmake")
 
 find_package(
-    RTIConnextDDS "6.0.0"
+    RTIConnextDDS "6.1.0"
     REQUIRED
     COMPONENTS
         core
         messaging_api
+        routing_service
 )
 
+# Select the adequate version of the Recording Service library depending on the
```

Comment:
This is something failing because of the Find Package version. Instead of this workaround, we need to find the root of the issue and fix it.

Reply:
Working on that.